client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
IIRC we had `isNewUser` implemented with `moment` at some point, and we eventually shifted back to this lower-level approach, _for some reason I can't recall_. (*)

(*) I think it was just that our particular `moment` solution was off

> This seems like something that should be in `state/current-user/selectors.js`

Yeah, this is always a hard problem. After we polish a few things, we'd like to organize our Guided Tours files a little better, and that includes sorting out these "context functions", we are really just selectors — though usually selectors that either cross data from different data branches, or with very specific business rules, or selectors that parse `guided-tours`/`action-log` data.

> y'know, we could go the whole global selectors route

It admittedly would make it easier here. :)